pipeline/data/importers.py

```diff
@@ -0,0 +1,221 @@
+import shutil
+import subprocess
+import tarfile
+from enum import Enum
+from pathlib import Path
+import zipfile
+
+
+from pipeline.common.command_runner import run_command
+from pipeline.common.downloads import stream_download_to_file, compress_file, DownloadException
+from pipeline.common.logging import get_logger
+
+logger = get_logger(__file__)
+
+
+class Importer(Enum):
+    opus = "opus"
+    mtdata = "mtdata"
+    sacrebleu = "sacrebleu"
+    flores = "flores"
+    url = "url"
+
+
+def opus(src: str, trg: str, dataset: str, output_prefix: Path):
+    """
+    Download a dataset from OPUS
+
+    https://opus.nlpl.eu/
+    """
+    logger.info("###### Downloading opus corpus")
```

Comment:
I'd prefer not to have these `####` marks in Python files. I'm assuming they were useful in bash files, but here I would prefer to rely on the loggers and their `[name] log message` format to disambiguate logs. Also this log is repeated twice.